src/components/views/read/cvForm/templates/FirstTemplate.tsx

```diff
@@ -0,0 +1,310 @@
+              .map(([key, value]) => (
+                <View key={key}>
+                  <Text style={{ fontSize: '11px' }}>
+                    * {key.charAt(0).toUpperCase() + key.slice(1)}: {value}
+                  </Text>
+                </View>
+              ))}
+          </View>
+        ))}
+        <Text>Employment History</Text>
+        <View
+          style={{
+            height: '1px',
+            marginTop: '13px',
+            marginBottom: '12px',
+            width: '100%',
+            backgroundColor: '#084c41',
+          }}
+        />
+        {workData.map((item, index) => (
+          <View key={index} style={{ marginBottom: 10 }}>
+            {Object.entries(item)
+              .filter(([key]) => key !== 'id') // Exclude the key "id"
+              .map(([key, value]) => (
+                <View key={key}>
+                  <Text style={{ fontSize: '11px' }}>
+                    * {key.charAt(0).toUpperCase() + key.slice(1)}: {value}
+                  </Text>
+                </View>
+              ))}
+          </View>
+        ))}
+        <Text>Skills</Text>
+        <View
+          style={{
+            height: '1px',
+            marginTop: '13px',
+            marginBottom: '12px',
+            width: '100%',
+            backgroundColor: '#084c41',
+          }}
+        />
+        {skillsData.map(
+          (item, index) =>
+            item.skills.length > 0 && (
+              <View key={index} style={{ marginBottom: 10 }}>
+                <Text style={{ fontSize: '12px' }}>
+                  * {item.title.charAt(0).toUpperCase() + item.title.slice(1)}:
+                </Text>
+                {item.skills.map((skill: string, skillIndex: number) => (
+                  <Text key={skillIndex} style={{ fontSize: '9px' }}>
+                    - {skill}
+                  </Text>
+                ))}
+              </View>
+            ),
+        )}
+
+        <Text>Portfolio & Projects</Text>
+        <View
+          style={{
+            height: '1px',
+            marginTop: '13px',
+            marginBottom: '12px',
+            width: '100%',
+            backgroundColor: '#084c41',
+          }}
+        />
+        {portfolioData.map((item, index) => (
+          <View key={index} style={{ marginBottom: 10 }}>
+            {Object.entries(item)
+              .filter(([key]) => key !== 'id') // Exclude the key "id"
+              .map(([key, value]) => (
+                <View key={key}>
+                  <Text style={{ fontSize: '11px' }}>
+                    * {key.charAt(0).toUpperCase() + key.slice(1)}: {value}
+                  </Text>
+                </View>
+              ))}
+          </View>
+        ))}
+        <Text>Self-Motivation</Text>
+        <View
+          style={{
+            height: '1px',
+            marginTop: '13px',
+            marginBottom: '12px',
+            width: '100%',
+            backgroundColor: '#084c41',
+          }}
+        />
```

Comment:
Refactor this to a component. And re-use it wherever you need a horizontal rule.

Reply:
fixed